### PR TITLE
[NEW] Add runtime provider management and support VM2

### PR DESCRIPTION
**Pull request by d-gubert**
# What? :boat:

Add runtime provider management and support VM2
# Why? :thinking:

TBD
# Links :earth_americas:

# PS :eyes:

### CODE REVIEW

**Review by pierre-lehnen-rc — APPROVED** [review]
I've reviewed only the videoconf changes.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rocket.chat/apps-engine",
-  "version": "1.35.0-alpha",
+  "version": "1.35.0-vm2",
```
Comment (d-gubert) [inline package.json:3]:
```suggestion
  "version": "1.35.0-alpha",
```

File: src/definition/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rocket.chat/apps-ts-definition",
-  "version": "1.33.0-alpha",
+  "version": "1.35.0-vm2",
```
Comment (d-gubert) [inline src/definition/package.json:3]:
```suggestion
  "version": "1.35.0-alpha",
```

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@rocket.chat/apps-engine",
-    "version": "1.35.0-alpha",
+    "version": "1.35.0-vm2",
```
Comment (d-gubert) [inline package-lock.json:3]:
```suggestion
    "version": "1.35.0-alpha",
```

**Review by AllanPazRibeiro — APPROVED** [review]

**Pull request merged**